eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -88,6 +90,46 @@ stages:
       useMonoRuntime: false              # 🚫
       withPreviousSDK: false             # 🚫
 
+  - template: ../jobs/vmr-build.yml
+    parameters:
+      # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
+      buildName: MarinerCrossX64_Online_MsftSdk
+      isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
+      vmrBranch: ${{ variables.VmrBranch }}
+      architecture: x64
+      pool:
+        name: ${{ variables.defaultPoolName }}
+        demands: ${{ variables.defaultPoolDemands }}
+      container: ${{ parameters.marinerx64CrossContainer }}
+      buildFromArchive: false            # 🚫
+      enablePoison: false                # 🚫
+      excludeOmniSharpTests: true        # ✅
+      runOnline: true                    # ✅
+      useMonoRuntime: true               # ✅
+      withPreviousSDK: false             # 🚫
+      crossRootFs: '/crossrootfs/x64'    # 📝
+      overrideRid: 'linux-x64'           # 📝
```

Comment:
Why do we need `overrideRid`? I though we standardized on the GNU configure terms `build`, `host` and `target`: https://gcc.gnu.org/onlinedocs/gccint/Configure-Terms.html#:~:text=So%2C%20build%20is%20the%20machine,you're%20building%20libraries).

In this case, `BuildRid`, `HostRid` and `TargetRid`

Reply:
It would be _lovely_ if we were using the GNU terms throughout.

But We pretty consistently use `TargetRid` throughout the project to _mean_ `HostRid`, and only one repo uses the term `HostRid`

In this case I've swapped `overrideRid` to match the wrong-but-pervasive `targetRid`